modules/custom/chatbot_migration/src/Plugin/migrate/source/Personas.php

```diff
@@ -0,0 +1,195 @@
+    $query->leftJoin('field_data_field_facebook', 'fdffb', 'fdffb.entity_id = n.nid AND fdffb.revision_id = n.vid');
+    $query->leftJoin('field_data_field_twitter', 'fdftw', 'fdftw.entity_id = n.nid AND fdftw.revision_id = n.vid');
+    $query->leftJoin('field_data_field_google_', 'fdfg', 'fdfg.entity_id = n.nid AND fdfg.revision_id = n.vid');
+    $query->leftJoin('field_data_field_linkedin', 'fdfli', 'fdfli.entity_id = n.nid AND fdfli.revision_id = n.vid');
+    $query->leftJoin('field_data_field_canal_youtube', 'fdfcy', 'fdfcy.entity_id = n.nid AND fdfcy.revision_id = n.vid');
+    $query->leftJoin('field_data_field_canal_vimeo', 'fdfcv', 'fdfcv.entity_id = n.nid AND fdfcv.revision_id = n.vid');
+    // For fields referencing taxonomies.
+    //$query->leftJoin('field_data_field_provincia_canton_distrito', 'fdfpcd', 'fdfpcd.entity_id = n.nid AND fdfpcd.revision_id = n.vid');
+    $query->leftJoin('field_data_field_autoria', 'fdfa', 'fdfa.entity_id = n.nid AND fdfa.revision_id = n.vid');
+    // For fields referencing locations.
+    //$query->leftJoin('field_data_field_location', 'fdfl', 'fdfl.entity_id = n.nid AND fdfl.revision_id = n.vid');
+    //$query->leftJoin('location', 'l', 'l.lid = fdfl.field_location_lid');
+
+    /* Add fields for each table.*/
+    // body.
+    $query->addField('fdb', 'body_value');
+    // field_direccion_detallada.
+    $query->addField('fdfdd', 'field_direccion_detallada_value');
+    // field_telefono_de_oficina.
+    $query->addField('fdftdo', 'field_telefono_de_oficina_value');
+    // field_telefono_habitacion.
+    $query->addField('fdfth', 'field_telefono_habitacion_value');
+    // field_telefono_movil.
+    $query->addField('fdftm', 'field_telefono_movil_value');
+    // field_fax.
+    $query->addField('fdff', 'field_fax_value');
+    // field_sitio_web.
+    $query->addField('fdfsw', 'field_sitio_web_url');
+    // field_correo_electronico.
+    $query->addField('fdfce', 'field_correo_electronico_email');
+    // field_correo_electronico_alt.
+    $query->addField('fdfcea', 'field_correo_electronico_alt_email');
+    // field_apartado_postal.
+    $query->addField('fdfap', 'field_apartado_postal_value');
+    // field_facebook.
+    $query->addField('fdffb', 'field_facebook_url');
+    // field_twitter.
+    $query->addField('fdftw', 'field_twitter_url');
+    // field_google_.
+    $query->addField('fdfg', 'field_google__url');
+    // field_linkedin.
+    $query->addField('fdfli', 'field_linkedin_url');
+    // field_canal_youtube.
+    $query->addField('fdfcy', 'field_canal_youtube_url');
+    // field_canal_vimeo.
+    $query->addField('fdfcv', 'field_canal_vimeo_url');
+    // field_autoria.
+    $query->addField('fdfa', 'field_autoria_tid');
+
+    $query->condition('n.type', 'fichas_personas');
+    $query->condition('n.status', 1);
+
+    return $query;
+  }
+
+  public function fields() {
+    $fields = [
+      'nid' => t('Node id.'),
```

Comment:
I'm not quite sure; but I think `$this->t()` should work in this context. Could you try it? Please 
If it works; that's the best way to do it